src/Store.ts

```diff
@@ -35,12 +96,39 @@ export class Store extends Evented {
 		return patchResult.undoOperations;
 	}
 
+	public at = <U = any>(path: Path<T, Array<U>>, index: number): Path<T, U> => {
+		const array: any[] = this.get(path);
```

Comment:
is this really an `any[]`?

Reply:
No it's not. The type annotation isn't needed at all. Good catch